typing/trigger.d.ts

```diff
@@ -0,0 +1,54 @@
+import { lang } from './global';
+import { Data } from './data';
+
+export interface Output {
+  [key: string]: () => string
+}
+
+interface Match {
+  target?: string
+  [key: string]: string
+}
+
+
+type LanguageText = Record<lang, string>
+type Response = (data: Data, matches: Match, output: Output) => string | LanguageText
```

Comment:
> I think overall it would be better to put this file in a separate pull request as there's a bunch of details here that need to be ironed out, and this file isn't used yet.

OK, I'll leave only `typings/data.d.ts` to see if ci running as expected, and move other type definition to a seprated pr